python/cudf/cudf/core/dtypes.py

```diff
@@ -531,6 +585,21 @@ def from_pandas(cls, pd_dtype: pd.IntervalDtype) -> "IntervalDtype":
             subtype=pd_dtype.subtype
         )  # TODO: needs `closed` when we upgrade Pandas
 
+    def serialize(self) -> Tuple[dict, list]:
+        header: Dict[str, Any] = {
```

Comment:
Doesn't really matter, but I'm surprised this type isn't automatically inferred.

Reply:
Turns out it was unnecessary, so dropped.